Sources/Amplitude/Amplitude.m

```diff
@@ -658,6 +660,10 @@ - (void)annotateEvent:(NSMutableDictionary *)event {
     [event setValue:library forKey:@"library"];
     [event setValue:[AMPUtils generateUUID] forKey:@"uuid"];
     [event setValue:[NSNumber numberWithLongLong:[self getNextSequenceNumber]] forKey:@"sequence_number"];
+    
```

Comment:
Since we have a new key `plan` in the event payload. Dose this change also depends on our backend team?

Reply:
@yuhao900914 yeah, I guess Jeffery will handle the backend change